L1Trigger/TrackTrigger/src/TTClusterAlgorithm_neighbor.cc

```diff
@@ -44,8 +44,8 @@ template< >
 bool TTClusterAlgorithm_neighbor< Ref_Phase2TrackerDigi_ >::isANeighbor( const Ref_Phase2TrackerDigi_& center,
                                                                  const Ref_Phase2TrackerDigi_& mayNeigh ) const
 {
-  unsigned int rowdist = abs(center->row() - mayNeigh->row());
-  unsigned int coldist = abs(center->column() - mayNeigh->column());
+  unsigned int rowdist = std::abs((int)(center->row()) - (int)(mayNeigh->row()));
```

Comment:
this is definitely a functionality change (to be correct) as row() and column() return unsigned ints. 
However it looks like this class is not used in CMSSW.